vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -0,0 +1,257 @@
+gcr.io/x-team)
+- vCluster Pro syncer (control plane `StatefulSet`) pod default resource requirements (may need to be increased based on intended vCluster workloads):
+
+```yaml title="vCluster Pro syncer pod resource requirements"
+  - Requests
+    - memory: 256Mi
+    - cpu: 200m
+    - ephemeral-storage: 400Mi
+  - Limits
+    - memory: 2Gi
+    - ephemeral-storage: 8Gi
+```
+
+### Download vCluster Helm chart
+
+Download https://charts.loft.sh/charts/vcluster-0.20.3.tgz:
+
+```bash title="Download vCluster Helm chart"
+curl -O https://charts.loft.sh/charts/vcluster-0.20.3.tgz
+```
+
+### Download and push required container images
+
+For clusters unable to pull images from Docker Hub, you need to push the platform images to your private registry. Each vCluster release includes a `vcluster-images.txt` file that lists the necessary images.
+
+:::warning
+When using virtual clusters in air-gapped environments, the
+`config.experimental.deploy.vcluster.helm` [configuration setting](/docs/vcluster/configure/vcluster-yaml/#experimental-deploy-vcluster) does not work with external Helm repositories since they cannot be accessed. This means custom Helm charts from repositories like `charts.bitnami.com` cannot be used for virtual cluster deployments.
+:::
+
+Follow these instructions to download all vCluster images and import them to your private registry:
+
+<Flow id="offline-images">
+    <Step>
+      Set the `VERSION` variable to the vCluster version you want to upgrade to or set it to the current version using:
+
+      ```bash title="Set VERSION variable"
+      CHART=$(kubectl get service vcluster -n vcluster -o jsonpath={.metadata.labels.chart})
+      VERSION=${CHART:5}
+      ```
+  </Step>
+  <Step>
+
+Set your private registry as a variable:
+
+```bash title="Set private registry"
+REGISTRY=ecr.io/myteam      # This should be a prefix; do not include any LOFT_IMAGE paths
+```
+
+  </Step>
+  <Step>
+
+Download the `loft-images.txt` file and the required scripts `download-images.sh` and `push-images.sh`, then make them executable:
+
+```bash title="Download and prepare scripts"
+wget https://github.com/loft-sh/vcluster/releases/download/v${VERSION}/vcluster-images.txt
+wget https://github.com/loft-sh/vcluster/releases/download/v${VERSION}/download-images.sh
+wget https://github.com/loft-sh/vcluster/releases/download/v${VERSION}/push-images.sh
+
+chmod +x ./download-images.sh
+chmod +x ./push-images.sh
+```
+
+  </Step>
+  <Step>
+
+Run `download-images.sh` to download all images locally:
+
+```bash title="Download images"
+./download-images.sh --image-list loft-images.txt
+```
+
+  </Step>
+  <Step>
+
+Run `push-images.sh` to push all downloaded images to your private registry:
+
+```bash title="Push images to private registry"
+./push-images.sh --registry ${REGISTRY}
+```
+
+    </Step>
+    <Step>
+
+Edit your existing `vcluster.yaml` file or create a new one with the following content:
```

Comment:
These are the incorrect values to set. These don't exist in the vcluster Helm chart. 

Here are the chart values: 
https://github.com/loft-sh/vcluster/blob/main/chart/values.yaml

```
controlPlane:
  advanced:
    defaultImageRegistry: $REGISTRY
  distro:
    k8s:
      version: $VERSION
```

Each individual version would need to be set for k0s and k3s on each component.

Reply:
Since only `k8s` distro is supported, we can just leave this snippet.